crates/fj/src/angle.rs

```diff
@@ -0,0 +1,101 @@
+use std::f64::consts::PI;
+
+/// An angle
+#[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature = "serialization", derive(Serialize, Deserialize))]
+pub struct Angle {
+    // The value of the angle in radians
+    rad: f64,
+}
+
+impl Angle {
+    /// Create a new angle specified in radians
+    pub fn from_rad(rad: f64) -> Self {
+        Self {
+            rad: Self::wrap(rad),
+        }
+    }
+    /// Create a new angle specified in degrees
+    pub fn from_deg(deg: f64) -> Self {
+        Self::from_rad(deg * PI / 180.)
+    }
+    /// Retrieve value of angle as radians
+    pub fn rad(&self) -> f64 {
+        self.rad
+    }
+    /// Retrieve value of angle as degrees
+    pub fn deg(&self) -> f64 {
+        self.rad / PI * 180.
```

Comment:
Can be simplified using [`f64::to_degrees`](https://doc.rust-lang.org/std/primitive.f64.html#method.to_degrees):
```suggestion
        self.rad.to_degrees()
```